cpp/dolfinx/geometry/BoundingBoxTree.h

```diff
@@ -266,7 +266,7 @@ class BoundingBoxTree
   /// build the bounding box tree for
   /// @param[in] padding Value to pad (extend) the the bounding box of
   /// each entity by.
-  BoundingBoxTree(const mesh::Mesh<T>& mesh, int tdim, T padding = 0)
```

Comment:
Could you switch this literal 0 to 0.0 so it's 100% clear T is a floating point type.

Reply:
That line was removed in a previous commit in order to make padding a mandatory argument.